package.json

```diff
@@ -1546,23 +1547,24 @@
                     "description": "Types to exclude from showing in the Interactive variable explorer",
                     "scope": "resource"
                 },
-                "jupyter.codeRegularExpression": {
-                    "type": "string",
-                    "default": "^(#\\s*%%|#\\s*\\<codecell\\>|#\\s*In\\[\\d*?\\]|#\\s*In\\[ \\])",
-                    "description": "Regular expression used to identify code cells. All code until the next match is considered part of this cell.",
-                    "scope": "resource"
-                },
-                "jupyter.defaultCellMarker": {
-                    "type": "string",
-                    "default": "# %%",
-                    "description": "Cell marker used for delineating a cell in a python file.",
-                    "scope": "resource"
-                },
-                "jupyter.markdownRegularExpression": {
-                    "type": "string",
-                    "default": "^(#\\s*%%\\s*\\[markdown\\]|#\\s*\\<markdowncell\\>)",
-                    "description": "Regular expression used to identify markdown cells. All comments after this expression are considered part of the markdown.",
-                    "scope": "resource"
+                "jupyter.codeLensExpressions": {
+                    "type": "array",
```

Comment:
Looks like this would trash existing user settings here? I think we might need a migration to ship this.

Reply:
Yeah that's probably true. I think I'll change it to support both. The old one could be for python. The new one would trump the old one if found.

Reply:
Should both be supported now.